modules/learn/pages/buckets-memory-and-storage/storage-settings.adoc

```diff
@@ -45,9 +45,19 @@ A starting-point for experimentation is to establish the numbers for reader thre
 
 See the _General-Settings_ information on xref:manage:manage-settings/general-settings.adoc#data-settings[Data Settings], for details on how to establish appropriate numbers of reader and writer threads.
 
+Note also that threads can also be configured for the _NonIO_ and _AuxIO_ thread pools:
+
+* The _NonIO_ thread pool is used to run _in memory_ tasks -- for example, the _durability timeout_ task.
+
+* The _AuxIO_ thread pool is used to run _auxiliary I/O_ tasks -- for example, the _access log_ task.
+
+Again, the maximum thread-allocation for each is _64_, the minimum _4_.
```

Comment:
The minimum is 1, if 0 or default is passed as an argument instead Memcached will calculate the number of threads needed by itself.

Reply:
I've modified. I've confined discussion of default to the rest page, which I think is okay, because we are summarising here.